### PR TITLE
Limit records per publish to cloudflare to 250

### PR DESCRIPTION
**Pull request by alanorwick**

### CODE REVIEW

File: dns_cloudflare.go [hunk 1]
```diff
@@ -107,8 +107,14 @@ func (c *cloudflareClient) checkZone(name string) error {
 func (c *cloudflareClient) uploadRecords(name string, records map[string]string) error {
 	// Convert all names to lowercase.
 	lrecords := make(map[string]string, len(records))
+	// Limit number of records to publish to cloudflare
+	i := 0
 	for name, r := range records {
+		if i >= 250 {
```
Comment (wizeguyy) [inline dns_cloudflare.go:113]: I don't understand this. `range` returns a (position, value) pair, e.g:  `i, v := range`. Why do you call the position `name`? Why do you need an extra `i` counter? Wouldn't this be less redundant and more concise?
```
for i, record := range records[..250] {
    lrecords[strings.ToLower(i)] = record
}
```

Comment (gameofpointers) [reply dns_cloudflare.go:113]: i agree

Comment (alanorwick) [reply dns_cloudflare.go:113]: You cannot slice a map like records[..250]. Maps in Go are not ordered, so there's no concept of the first 250 items in a map.

Comment (alanorwick) [reply dns_cloudflare.go:113]: second to that, it's called name rather than i since name is the index in the map.

Comment (gameofpointers) [reply dns_cloudflare.go:113]: oh my bad, i didn't see that its a map
change looks good to me then

**Review by gameofpointers — APPROVED** [review]

**Pull request merged**